src/styles/utils/_varibles.scss

```
@@ -0,0 +1,9 @@
+$calendarWidth: 707px;
```

calculate this value 
```suggestion
$calendarWidth: some calculations ..;
```